app/src/lib/git/stash.ts

```diff
@@ -216,60 +225,3 @@ function extractBranchFromMessage(message: string): string | null {
   const match = desktopStashEntryMessageRe.exec(message)
   return match === null || match[1].length === 0 ? null : match[1]
 }
-
-/**
- * Get the files that were changed in the given stash commit.
- *
- * This is different than `getChangedFiles` because stashes
- * have _3 parents(!!!)_
- */
-export async function getStashedFiles(
```

Comment:
If we remove this method won't that mean we'll break the display of any existing stashes that contain untracked files? Could we perhaps keep this around for the time being and consider moving to the simpler approach at a later date?

We could also potentially figure out whether the stash entry includes untracked files ahead of time (from `%P` in the `log` call) but that seems a tad too complicated compared to just keeping this method around for now.

Reply:
oh yes that is a super good point @niik

Reply:
restored in 3651a9f !

Reply:
actually restored in 6b9604e and 9f2caa6dff512fcfdc62bb0fd5407b285f25ac25